example.tex

```diff
@@ -31,29 +37,29 @@
 \chapter{Chapter 1: The Dark \LaTeX}
 
 \section{Main Section}
-\lipsum[1] % filler text
+\lipsum[2] % filler text
+
+\begin{quotebox}
+	As you approach this template you get a sense that the blood and tears of many generations went into its making. A warm feeling welcomes you as you type your first words.
+\end{quotebox}
 
 \subsection{Fun with boxes}
 \subsubsection{Even more fun!}
 
-\lipsum[1-2]
-
 \begin{commentbox}{This Is a Comment Box!}
-  A commentbox is a box for minimal highlighting of text. It lacks the ornamentation of paperbox, but it can handle being broken over a column.
-  
-  \lipsum[2]
+  A \lstinline!commentbox! is a box for minimal highlighting of text. It lacks the ornamentation of \lstinline!paperbox!, but it can handle being broken over a column.
 \end{commentbox}
 
 \subtitlesection{Weapon, +1, +2, or +3}
 {Weapon (any), uncommon (+1), rare (+2), or very rare (+3)}
 
-\lipsum[2]
-
-\begin{quotebox}
-	As you approach this template you get a sense that the blood and tears of many generations went into its making. A warm feeling welcomes you as you type your first words.
+\lipsum[3]
 
-	\lipsum[2]
-\end{quotebox}
+\begin{figure}[!t]
+	\begin{paperbox}{Behold\, the Paperbox!}
```

Comment:
Why the extra thin space here?

Reply:
Ah, thanks. You made me realize I handled an error thrown by including the comma in the tcolorbox title. I had to surround the second parameter with curly braces to accept the title. I will make the same fix to the other uses of parameters in the newtcolorbox implementations.